src/main/java/com/sugarcrm/candybean/automation/Candybean.java

```diff
@@ -52,7 +54,13 @@
 	/**
 	 * The default configuration file name to instantiate candybean.
 	 */
-	public static final String DEFAULT_CONFIG_FILE = ROOT_DIR + File.separator + "candybean.config";
```

Comment:
Since VoodooGrimoire loads the runner before it loads the Candybean config file, the runner doesn't know where to look for the config file in Grimoire. I've added multiple possibilities for where the config file could be, and candybean will use whichever one it finds first.

The alternate way to do this would be to pass the config path using an environment variable, a la
```
mvn -Dcandybean.config="PATH/TO/CONFIG"
```
but I'm not sure I like having to start Voodoo runs with with an environment variable every time. Additionally, this would require additional configuration steps in Intellij/Eclipse to properly run the tests.